rice-framework/krad-sampleapp/web/pom.xml

```diff
@@ -38,7 +38,7 @@
     <jmeter.excludeJmeterTestFile>**/KRAD.jmx</jmeter.excludeJmeterTestFile>
     <kuali.testjar.phase>none</kuali.testjar.phase>
     <kuali.generate.reports>false</kuali.generate.reports>
-    <webapp.working.dir>${project.build.directory}/${project.artifactId}-${project.version}</webapp.working.dir>
+    <webapp.working.dir>${project.build.directory}/${build.finalName}</webapp.working.dir>
```

Comment:
build.finalName is a reserved Maven property that is always set.  This corresponds to an entry in the top level pom.xml that simplifies the way artifacts are named locally.  

`<build>
  <finalName>${project.artifactId}</finalName>
</build>`